.github/actions/version_checking/action.yml

```diff
@@ -0,0 +1,54 @@
+name: version_checking
```

Comment:
Together with tarantool/actions maintainer we decided to use `check-module-version ` name

Reply:
Done, now it is `.github/actions/check-module-version/action.yml` with `name: Check module version ` inside